pss/outbox/outbox.go

```diff
@@ -96,21 +91,15 @@ func (o *Outbox) Stop() {
 
 // Enqueue a new element in the outbox if there is any slot available.
 // Then send it to process. This method is blocking if there is no workers available.
-func (o *Outbox) Enqueue(outboxMsg *outboxMsg) error {
+func (o *Outbox) Enqueue(outboxMsg *outboxMsg) {
 	// first we try to obtain a slot in the outbox.
+	slot := <-o.slots
```

Comment:
This should be a select with return on o.stopC. It is a potential deadlock.

Reply:
Fixed